crates/libwebrtc-sys/src/bridge.rs

```diff
@@ -469,6 +686,69 @@ pub(crate) mod webrtc {
             error: &CxxString,
         );
 
+        /// Completes the provided [`DynPeerConnectionEventsHandler`].
+        pub fn on_signaling_change(
+            cb: &mut DynPeerConnectionEventsHandler,
+            state: SignalingState,
+        );
+
+        /// Completes the provided [`DynPeerConnectionEventsHandler`].
+        pub fn on_standardized_ice_connection_change(
+            cb: &mut DynPeerConnectionEventsHandler,
+            new_state: IceConnectionState,
+        );
+
+        /// Completes the provided [`DynPeerConnectionEventsHandler`].
+        pub fn on_connection_change(
```

Comment:
@rogurotus here all the docs are same, which isn't quite useful. You should describe for the each function in what circumstances it completes the handler.